config.py

```diff
@@ -30,20 +30,45 @@ class Config(object):
     PASSWORD_MIN_LENGTH = 8
     PASSWORD_MAX_LENGTH = 160
 
+    RM_CASE_SERVICE_HOST = os.getenv('RM_CASE_SERVICE_HOST', 'localhost')
+    RM_CASE_SERVICE_PORT = os.getenv('RM_CASE_SERVICE_PORT', 8171)
+    RM_CASE_SERVICE_PROTOCOL = os.getenv('RM_CASE_SERVICE_PROTOCOL', 'http')
+    RM_CASE_SERVICE = f'{RM_CASE_SERVICE_PROTOCOL}://{RM_CASE_SERVICE_HOST}:{RM_CASE_SERVICE_PORT}'
```

Comment:
We are moving away from this approach now it should just be CASE_SERVICE_URL. The host, port and protocol use are being discontinued and  RAS or RM in front of service URL names is not being used anymore.

Reply:
@gemmai95 Oh, nice! Have we got that proposition documented somewhere?

Reply:
Unfortunately not just word of mouth atm.

Reply:
@gemmai95 I'll see about getting it in this PR anyway 👍

Reply:
Example in r-ops found [here](https://github.com/ONSdigital/response-operations-ui/blob/0e2b966d637af71a34d7879b15af53e7a7ead2df/config.py#L33).